tests/e2e/vsan_stretched_cluster.go

```diff
@@ -320,16 +342,24 @@ var _ = ginkgo.Describe("[vsan-stretch-vanilla] vsan stretched cluster tests", f
 	   8.  Delete statefulsets and its pvcs created in step 2
 	   9.  Bring primary site up and wait for testbed to be back to normal
 	*/
-	ginkgo.It("[primary-centric][control-plane-on-primary] Statefulset scale up/down while primary"+
+	ginkgo.It("[primary-centric][control-plane-on-primary] Statefulset scale up-down while primary"+
 		" site goes down", func() {
 		ctx, cancel := context.WithCancel(context.Background())
 		defer cancel()
+
 		ginkgo.By("Creating StorageClass for Statefulset")
-		// decide which test setup is available to run
+		//decide which test setup is available to run
 		ginkgo.By("CNS_TEST: Running for vanilla k8s setup")
 		scParameters = map[string]string{}
-		scParameters["StoragePolicyName"] = storageThickPolicyName
-		storageClassName = "nginx-sc-thick"
+
+		if rwxAccessMode {
```

Comment:
give a proper comment as what this if/else is doing

Reply:
And i would suggest, since you have used this if/else block at multiple places, move this code to before each block

Reply:
done